columnflow/columnar_util.py

```diff
@@ -1554,7 +1558,13 @@ def get_dependencies(self, include_others: bool = False) -> set[ArrayFunction |
 
         return deps
 
-    def _get_columns(self, io_flag: IOFlag, call_cache: set | None = None, debug=False) -> set[str]:
+    def _get_columns(
+        self,
+        io_flag: IOFlag,
+        call_cache: set | None = None,
+        debug: bool = False,
```

Comment:
Sorry for piggybacking on this PR, but since you touched this line, could you remove the `debug` argument? This looks like some dev remnant that I left there a while ago :)

Reply:
Sure, no problem.